CHANGELOG.md

```diff
@@ -5,6 +5,9 @@ All notable changes to this project will be documented in this file.
 ## Added
 - Alias available in node metrics 
 
+## Changed
+- Adjusted KeyStoreUtils for CoMakery integration
```

Comment:
Please note here as well (it can be in a separate row) that you removed spongycastle and changed it to bouncycastle

Reply:
Done!